api/source/danger/danger_runner.ts

```diff
@@ -102,9 +109,9 @@ export async function runDangerAgainstFileInline(
   const context = contextForDanger(dangerDSL)
   const dangerRuntimeEnv = await exec.runner.createDangerfileRuntimeEnvironment(context)
 
-  if (dangerRuntimeEnv.sandbox) {
-    // Mutates runtimeEnv.sandbox
-    await appendPerilContextToDSL(installationRun.iID, undefined, dangerRuntimeEnv.sandbox, peril)
+  if (dangerRuntimeEnv) {
+    // Mutates runtimeEnv
+    await appendPerilContextToDSL(installationRun.iID, undefined, dangerRuntimeEnv, peril)
```

Comment:
This is actual fix - rest of the PR is either forced prettier re-formatting or very minor (not critical) adjustments

Reply:
It's fine, you can add the param `?w=1` to the PR to get whitespace changes removed from the diff, which made it easier to review